modules/ROOT/pages/deployment/services/s-list/search.adoc

```diff
@@ -69,6 +69,8 @@ When using the Tika container and docker-compose, consider the following:
 * See the xref:deployment/container/orchestration/orchestration.adoc#docker-compose-examples[Docker Compose Examples] (ocis_wopi) for details.
 * Containers for the linked service are reachable at a hostname identical to the alias or the service name if no alias was specified.
 
+When using the Tika extractor, make sure to also set `FRONTEND_FULL_TEXT_SEARCH_ENABLED` in the xref:{s-path}/frontend.adoc[frontend service] to `true`. This will tell the webclient that full-text search has been enabled.
```

Comment:
```suggestion
When using the Tika extractor, make sure to also set `FRONTEND_FULL_TEXT_SEARCH_ENABLED` in the xref:{s-path}/frontend.adoc[frontend service] to `true`. This will tell the web client that full-text search has been enabled.
```